app/src/html/toolinfo.json

```diff
@@ -1,21 +1,20 @@
 {
+  "$schema": "/toolinfo/1.2.0-draft02",
+  "$language": "en",
   "name": "iabot",
-  "title": "IABot Management Interface",
-  "subtitle": "The front-end control panel to configuring and directing InternetArchiveBot's activities.",
-  "description": "This is a tool where users can easily find and repair dead links on supported wikis.  They can also queue InternetArchiveBot up onto a list of pages, help correct erroneous entries in the DB, and report bugs.",
-  "keywords": "iabot, internetarchivebot, management, interface, linkrot, bot, dead, link, fixing",
+  "title": "InternetArchiveBot Management Interface",
```

Comment:
You sure you want to expand this?  This may make the menubar needlessly large.

Reply:
Menu bar in which context?

Reply:
It's the big item in the top left of the UI.  If you expand this, it will make the menu bar require more screen real estate.  I'm trying to keep this mobile friendly.